File: assignment 10 - C# practice problems/15.cs
//Write a c# program to test whether a string end with "Script".
// The string length must be greater or equal to 6.
using System;
namespace MyNameSpace
{
    class Program15
    {
        public static void TestStringEndsWithScriptDriverFunction()
        {

            Console.WriteLine("Enter a string having length 6 or greater");
            string stringInput1 = Console.ReadLine();

            stringInput1 = stringInput1.Substring(stringInput1.Length - 6, 6);

            if (stringInput1 == "Script")
            {

                Console.WriteLine("String Ends With \"Script\"");

            }
            else
            {
                Console.WriteLine("String Doesn't End With \"Script\"");
            }


        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/15.js
//Write a JavaScript program to test whether a string end with "Script".
// The string length must be greater or equal to 6.

function testStringEndsWithScriptDriverFunction() {
    let stringInput = prompt("Enter a string having length 6 or greater");


    if (stringInput != null) {
        stringInput = stringInput.trim();

        stringInput = stringInput.substr(stringInput.length - 6, stringInput.length - 1);

        if (stringInput === 'Script') {

            alert("String Ends With \"Script\"");

        } else {
            alert("String Doesn't End With \"Script\"");
        }

    }
}

testStringEndsWithScriptDriverFunction()